src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -239,14 +237,14 @@ private InstructorCourseDetailsPage getCourseDetailsPage() {
         return loginAdminToPage(detailsPageUrl, InstructorCourseDetailsPage.class);
     }
 
-    private boolean didStudentReceiveReminder(String courseName, String courseId, String studentEmail,
-                                              String studentPassword)
+    // TODO replace with GAE LocalMailService stub as we are checking the logic here, not reliability of email delivery
```

Comment:
The test is run using test Gmail accounts, so that possibility is reasonably small. You are welcome to fix it if you want, though I think that'd be better suited to a separate issue/PR.